src/features/header/index.tsx

```diff
@@ -36,13 +36,13 @@ const Logo = styled.img`
 `
 
 type ButtonProps = {
-    color: string,
-    backgroundColor: string
+    color?: string,
+    backgroundColor?: string
 }
 
 const Button = styled.button<ButtonProps>`
-    background-color: ${(props: ButtonProps) => props.backgroundColor};
-    color: ${(props: ButtonProps) => props.color};
+    background-color: ${(props: ButtonProps) => props.backgroundColor ? props.backgroundColor : 'rgba(0,0,0,0)'};
```

Comment:
again, maybe try to move it into a separate CSS file?

Reply:
This is a styled component.